ios/Capacitor/Capacitor/Plugins/Camera.swift

```diff
@@ -232,7 +234,7 @@ public class CAPCameraPlugin: CAPPlugin, UIImagePickerControllerDelegate, UINavi
         }
 
         if settings.shouldResize {
-            guard let convertedImage = resizeImage(image!) else {
+            guard let convertedImage = resizeImage(image!, false) else {
```

Comment:
@dwieeb this should consider the flag from the setting (`settings.preserveAspectRatio`), right?

Reply:
yeah, good catch, should be
```suggestion
            guard let convertedImage = resizeImage(image!, settings.preserveAspectRatio) else {
```

Reply:
Thanks, these were not easy conflicts to resolve. 😂